Patryk_Fraczek/Device/ZadanieAngleCalculation.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of the ZadanieAngleCalculation project
+#
+# 
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+""" Calculate angle from double_scalar
+
+double_scalar from Tango_test to angle
+"""
+
+__all__ = ["ZadanieAngleCalculation", "main"]
+
+# PyTango imports
+import PyTango
+from PyTango import DebugIt
+from PyTango.server import run
+from PyTango.server import Device, DeviceMeta
+from PyTango.server import attribute, command
+from PyTango.server import class_property, device_property
+from PyTango import AttrQuality, AttrWriteType, DispLevel, DevState
+# Additional import
+# PROTECTED REGION ID(ZadanieAngleCalculation.additionnal_import) ENABLED START #
+from numpy import arcsin
+# PROTECTED REGION END #    //  ZadanieAngleCalculation.additionnal_import
+
+
+class ZadanieAngleCalculation(Device):
+    """
+    double_scalar from Tango_test to angle
+    """
+    __metaclass__ = DeviceMeta
+    # PROTECTED REGION ID(ZadanieAngleCalculation.class_variable) ENABLED START #
+    attr_proxy = PyTango.AttributeProxy('sys/tg_test/1/double_scalar')
+    value = attr_proxy.read().value
+    out = attr_proxy.read().value
+    # PROTECTED REGION END #    //  ZadanieAngleCalculation.class_variable
+    # ----------------
+    # Class Properties
+    # ----------------
+
+    # -----------------
+    # Device Properties
+    # -----------------
+
+    SecondDeviceName = device_property(
+        dtype='str', default_value="sys/tg_test/1/double_scalar"
+    )
```

Comment:
Mimo, iż zrobiłeś właściwość z nazwą urządzenia do odczytania atrybutu, to nigdzie z niej nie korzystasz w kodzie.
Zamiast tego wpisałeś na sztywno nazwy - to zła praktyka programistyczna.